dfindexeddb/indexeddb/cli.py

```diff
@@ -27,7 +27,7 @@
 from dfindexeddb.leveldb import record as leveldb_record
 from dfindexeddb.indexeddb.chromium import record as chromium_record
 from dfindexeddb.indexeddb.chromium import v8
-
```

Comment:
Should there be a two line gap here?

Reply:
good point, there shouldn't but the changes in this PR rectify this.